src/main/resources/liquibase/2.1.x.x/user_manual_translation.xml

```diff
@@ -13,7 +13,7 @@
 			<sqlCheck  expectedResult="0">SELECT count(*) FROM clinlims.menu WHERE element_id = 'menu_help_user_manual' AND action_url = '/docs/UserManual'</sqlCheck> 
 		</preConditions>
 		<update schemaName="clinlims" tableName="menu">
-			<column name="action_url" value="/docs/UserManual" />
+			<column name="action_url" value="/documentation/OEGlobal_UserManual_en.pdf" />
```

Comment:
definately we dont dont change existing liquibase changes sets. 
That breaks the existing installations. 
You just need to create a new changeset that does this update